stbt_rig.py

```diff
@@ -458,8 +458,15 @@ def cmd_run_prep(args, portal):
 def cmd_run_body(args, node, j):
     logger.info("Running tests...")
 
+    try:
+        root = find_test_pack_root()
+        test_cases = [os.path.relpath(x, root).replace('\\', '/')
+                      for x in args.test_cases]
+    except NotInTestPack:
+        test_cases = args.test_cases
```

Comment:
> We still support running tests outside a test-pack - we fallback to taking the test names verbatim.

Did we actually support running tests from (cwd) outside a test-pack? `cmd_run` calls `TestPack()` which calls `find_test_pack_root` which raises.

Aahhh but only in interactive mode. We do support running tests from outside a test-pack, in non-interactive mode.